src/service/mutator-impl.js

```diff
@@ -211,6 +212,12 @@ export class MutatorImpl {
         const ampElements = element.getElementsByClassName('i-amphtml-element');
         for (let i = 0; i < ampElements.length; i++) {
           const r = Resource.forElement(ampElements[i]);
+          userAssert(
```

Comment:
This will get logged into the user errors bucket, which is throttled at a higher rate. Instead, use `dev().error`.

Reply:
is `devAssert` fine?

Reply:
`devAssert` gets eliminated from production builds.

Reply:
Done